index2.html

```diff
@@ -309,10 +309,37 @@ <h3>Skills</h3>
 
 
     <footer>
-        <p>francksefu1998@gmail.com</p>
-        <div class="contentline">
+        <p class="mobile">francksefu1998@gmail.com</p>
+        <div class="contentline mobile">
             <div class="line"></div>
         </div>
+        <!--Desktop-->
+
+        <section class='icon desktop'>
+            <hr>
+            <ul>
+                <li>
+                    <a href="https://medium.com/@francksefu1998">
+                        <i class="fab fa-medium"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/franck_sefu">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/franck-sefu-884705254/">
+                        <i class="fab fa-linkedin-in"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/francksefu">
+                        <i class="fab fa-github"></i>
+                    </a>
+                </li>
+            </ul>
```

Comment:
- [ ] The Figma design proves that there is an icon that you are missing in your design. This icon should be included in the footer to stick to the Figma design and have a professional website. :+1: 
- Your design :sunglasses: 
![Screenshot from 2023-01-20 06-38-05](https://user-images.githubusercontent.com/99394374/213612798-98ae7541-e80f-4e89-951f-a128f908c858.png)

- Figma design :sunglasses: 
![Screenshot from 2023-01-20 06-38-51](https://user-images.githubusercontent.com/99394374/213612802-77fb943c-cee9-4a37-af05-086c1864e842.png)